src/inet/http_server.c

```diff
@@ -138,7 +145,7 @@ static int read_request(struct http_server_t *self_p,
                         struct http_server_request_t *request_p)
 {
     int res;
-    char buf[128];
```

Comment:
What's the reason for increasing the buffer size? As it is an embedded system we must always try to minimize the memory usage.